### PR TITLE
README.md: Add link to manual and make API reference link more prominent

### PR DESCRIPTION
**Pull request by bkeryan**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Add link to MeasurementLink manual.

Make API reference link more prominent. 

### Why should this Pull Request be merged?

Make documentation easier to find.

### What testing has been done?

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +34,13 @@ MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python fr
 
 ---
 
+## Documentation
+
+- [MeasurementLink Manual](https://www.ni.com/docs/en-US/bundle/measurementlink)
```
Comment (bkeryan) [inline README.md:39]: @nstokes-nati Is this the right terminology? And is this a stable URL that won't change?

**Review by DelpireNI — APPROVED** [review]

**Review by pbirkhol-ni — APPROVED** [review]

**Comment by bkeryan** [comment]
> This might be better for another PR, but I think we should get rid of the dependencies section. I was helping Jason earlier today get python set up on his machine, and the first thing he did after installing python was to manually install each dependency in that list, which is unnecessary since poetry manages dependencies for you. Also, the list in the README is almost certainly going to go out of date the next time we add a dependency.

I agree we should reconsider the dependencies section.

I also agree it would be better for another PR :)

File: README.md (hunk 1 above)
Comment (nstokes-nati) [reply README.md:39]: Yes, that's fine terminology. And yes, that's the best format for a direct link and I don't think this really merits a redirect.

**Review by nstokes-nati — APPROVED** [review]
Yeah, that's the best for a direct link and I don't think this really merits a redirect when we can just update the readme if we need to

**Pull request merged**